src/transition_tracer/transition_tracer.cpp

```diff
@@ -489,4 +505,53 @@ TransitionTracer::~TransitionTracer()
 {
 }
 
-} // namespace BSMPT
\ No newline at end of file
+void TransitionTracer::CheckMassRatio(const user_input &input,
+                                      const std::vector<double> &vec,
+                                      const double &temp) const
+{
+  std::stringstream ss;
+  std::vector<double> massOverTempSq, tmp;
```

Comment:
Maybe change these names ```massOverTempSq, tmp``` to others more meaningful? 
Like ```massOverTempHiggs, massOverTempGauge```?

Reply:
I renamed `tmp` into `massOverTempSqGauge`, as this is what it is. But kept the `massOverTempSq` because in the next step, we insert `massOverTempSqGauge` in it to have all thermal masses squared in one vector. Better?

Reply:
Yes, thank you ^^